client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -16,6 +16,36 @@ import { withSelect } from '@wordpress/data';
 import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 import { DEFAULT_POSTS, MAX_POSTS_TO_SHOW } from './constants';
 
+class MockupPostEdit extends Component {
+	render() {
+		const { displayDate, displayContext, displayThumbnails } = this.props;
+		const previewClassName = 'related-posts__preview';
+
+		return (
+			<div className={ `${ previewClassName }-post` }>
+				{ displayThumbnails && (
+					<Button className={ `${ previewClassName }-post-image-mockup` } isLink>
+						<span className={ `${ previewClassName }-post-image-mockup-text` }>Featured Image</span>
+					</Button>
+				) }
+				<h4>
+					<Button className={ `${ previewClassName }-post-link` } isLink>
+						{ __( 'Previews are currently unavailable in the editor' ) }
+					</Button>
+				</h4>
+				{ displayDate && (
+					<span className={ `${ previewClassName }-post-date has-small-font-size` }>
+						{ __( 'August 3, 2018' ) }
```

Comment:
I think we generally want to avoid surrounding mock-strings in translate calls so that we don't accidentally waste time translating them. Is this a mockup in the sense that it's used as a dummy preview for editors/authors or in the sense that it should disappear before merging into production?

Reply:
This is a mockup for a date that needs to be localisable. There is probably a better way to do this, but note that there are still inconsistencies between WP Core and Gutenberg in regards to the localisation of dates and I do think this is the best way to do this.

Reply:
I mean — I could probably generate a date each time and localise it. This is simply the most straightforward way and what was used here originally.

Reply:
I'll backup because I think I miscommunicated. It was hard to determine if this code was going to make it into production or was sitting here while we waited for "real" code to replace it.

That is, if `August 3, 2018` belongs in the final product as a fixed time string then this is fine. If it's going to be replace before merging into `master` then we probably want to remove the `__()` call.

Alternatively, if our goal is to have a localized date we should have functions to produce those and even with inconsistencies it's probably less of a burden on the translation teams to use localized date formatters than to ask people to translate specific points in time.

poking @akirk for his wisdom on this one

Reply:
This is basically the same date as was there already. i18n on dates in Gutenberg is a complex beast that I would love to avoid.